qa/L0_backend_config/test.sh

```diff
@@ -150,6 +152,136 @@ for ((i=0; i < ${#NEGATIVE_PARSE_ARGS[@]}; i++)); do
     fi
 done
 
+
+#
+# Sepcific backend tests
+# 
+
+# Tensorflow 1: Batching ON
+rm -rf ./models/
+mkdir -p ./models/no_config
+cp -r /data/inferenceserver/${REPO_VERSION}/qa_model_repository/savedmodel_float32_float32_float32/1 ./models/no_config/
+
+SERVER_ARGS="--backend-config=tensorflow,default-max-batch-size=5 $COMMON_ARGS"
+SERVER_LOG=$SERVER_LOG_BASE.backend_config_tensorflow_batch_5.log
+run_server
+
+if [ "$SERVER_PID" == "0" ]; then
+    echo -e "*** FAILED: Server failed to start $SERVER\n"
+    RET=1
+
+else
+    kill $SERVER_PID
+    wait $SERVER_PID
+
+    # Assert the max-batch-size is the command line value
+    DYNAMIC_BATCHING_LOG_LINE=$(grep "\"max_batch_size\": 5" $SERVER_LOG)
```

Comment:
Instead of checking server log, it would be more formal to use [model config protocol](https://github.com/triton-inference-server/server/blob/main/docs/protocol/extension_model_configuration.md#httprest) to get the model config

Reply:
Updated the tests to parse the returned model configuration rather than parsing the log.